lib/StoryblokComponent.vue

```diff
@@ -1,12 +1,39 @@
 <template>
-  <component :is="blok.component" v-bind="{ ...$props, ...$attrs }"></component>
+  <component :is="componentName" v-bind="{ ...$props, ...$attrs }"></component>
 </template>
 
 <script setup lang="ts">
-import type { SbBlokData } from "./types";
+import { ref, resolveDynamicComponent, inject } from "vue";
+import type { SbBlokData, SbVueSDKOptions } from "./types";
 
 export interface SbComponentProps {
   blok: SbBlokData;
 }
-defineProps<SbComponentProps>();
+const props = defineProps<SbComponentProps>();
+const componentFound: boolean =
+  typeof resolveDynamicComponent(props.blok.component) !== "string";
+
+// Fallback component logic
+const VueSDKOptions: SbVueSDKOptions = inject("VueSDKOptions");
+
+const componentName = ref(props.blok.component);
+if (!componentFound) {
+  if (!VueSDKOptions.enableFallbackComponent) {
+    console.error(
+      `Component could not be found for blok "${props.blok.component}"! Is it defined in main.ts as "app.component("${props.blok.component}", ${props.blok.component});"?`
+    );
+    /* throw new Error(
```

Comment:
Please, @manuelschroederdev can you review if the `thrown new Error` is the best option for this or not? I think `console.error` doesn't break the JS thread and also helps the developer to know what happened, isn't it?

Reply:
As discussed, please better `console.error` to don't block the JS thread